store/sdk/src/test/java/org/apache/carbondata/sdk/file/CarbonReaderTest.java

```diff
@@ -1522,4 +1522,208 @@ public boolean accept(File dir, String name) {
       e.printStackTrace();
     }
   }
+
+   @Test
+  public void testReadNextRowWithRowUtil() {
```

Comment:
As class not required. Test cases are not required for it

Reply:
If has cast type method, we can remove